SpatialConstAffine.lua

```diff
@@ -0,0 +1,39 @@
+local SpatialConstAffine, parent = torch.class('inn.SpatialConstAffine', 'nn.Module')
+
+function SpatialConstAffine:__init(nOutputPlane, inplace)
+  parent.__init(self)
+  self.a = torch.Tensor(1,nOutputPlane,1,1)
```

Comment:
it would be great to have a less hidden way of initialising `a` and `b`.
Maybe a method that takes a `SpatialBatchNormalization` and extract the relevant fields and set `a` and `b` to point to them ?